Resources/views/line/blog_post.html.twig

```diff
@@ -1,7 +1,7 @@
 {# In order to use blocks and override ez_render_field's template within this file,
  # we need to extend from another template and surround our content with a block.
  #}
-{% extends viewbaseLayout %}
+{% extends 'EzPublishCoreBundle::viewbase_layout.html.twig' %}
```

Comment:
It is. And I can confirm that it still occurs if I revert this patch.

Trying to investigate.